bokehjs/src/lib/models/tools/actions/help_tool.ts

```diff
@@ -32,7 +32,7 @@ export class HelpTool extends ActionTool {
     this.prototype.default_view = HelpToolView
 
     this.define<HelpTool.Props>(({String}) => ({
-      redirect: [ String, "https://docs.bokeh.org/en/latest/docs/user_guide/tools.html"],
+      redirect: [ String, "https://docs.bokeh.org/en/latest/docs/reference/models/tools.html"],
```

Comment:
This https://docs.bokeh.org/en/latest/docs/user_guide/interaction/tools.html is the correct link. Same in other cases.